docs/tutorials/fitting.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
<p>Thanks for this PR @AtreyeeS&nbsp;</p><p> </p><p>concerning how to select a thumbnail pic and add a tooltip for the gallery, I would recommend to open the notebook in a text editor, and edit the internal code there. It may sound risky, but it is much simpler. Find the code cell that produces&nbsp;the figure that you  would like for the gallery (for example this one :), and then replace the  --- "metadata": {}, ---  bit above the code cell with the snippet below:</p><p> </p><pre class="ql-syntax" spellcheck="false">"metadata": {
 "nbsphinx-thumbnail": {
  "tooltip": "Learn how to do perform a Fit in gammapy."
 }
</pre><p> </p><p>Note that you may write whatever you like after "tooltip"</p><p> </p><p>I will write all this process in the "Developer HowTo" of the docs, still missing.. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>